app/build.gradle

```
@@ -14,10 +14,10 @@ android {
     defaultConfig {
         applicationId "org.schabi.newpipe"
         resValue "string", "app_name", "NewPipe"
-        minSdk 19
-        targetSdk 29
-        versionCode 981
-        versionName "0.21.15"
+        minSdkVersion 19
+        targetSdkVersion 29
```

These changes should not be reverted (see https://github.com/TeamNewPipe/NewPipe/pull/7678): 
```suggestion
        minSdk 19
        targetSdk 29
```